### PR TITLE
 new update with title.

### PR DESCRIPTION
**Pull request by centrric-cntc0006**
This update just makes the user able to add a title at the top of the data table. This will be helpful for the end user he just has to import the CSV file to an EXCEL file, there he can easily merge the top cells to center this heading.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 node_modules/
 dist/
 example/package-lock.json
+example/node_modules/
```
Comment (dolezel) [inline .gitignore:6]: This is redundant
On line 3 it is defined without leading slash, so it matches node_modules in any subfolder

File: src/lib/csv.ts [hunk 2]
```diff
@@ -136,6 +138,8 @@ export default async function csv({
       if (!noHeader) {
         const headerNames = order.map((id) => map[id])
         if (headerNames.length > 0) {
+          if(title!=='')
```
Comment (dolezel) [inline src/lib/csv.ts:141]: Strange, this passed lint :/

**Comment by dolezel** [comment]
Also when adding new options, please update the README.md file

File: src/lib/csv.ts (hunk 2 above)
Comment (dolezel) [reply src/lib/csv.ts:141]: Fixed in #388

Comment (centrric-cntc0006) [reply src/lib/csv.ts:141]: Oh, can you explain why it shouldn't have passed?

**Comment by centrric-cntc0006** [comment]
I have updated the files.

File: src/lib/csv.ts (hunk 2 above)
Comment (dolezel) [reply src/lib/csv.ts]: There is setup lint and prettier config for consistent formatting, but there was an issue that it was not configured correctly and lint only disabled formatting rules, but didn't used rules from prettier

**Comment by dolezel** [comment]
Published in 3.1.0
Thanks for contributing!

**Pull request merged**